storybook/stories/ui-charts/manchetteWithSpaceTimeChart/base.stories.tsx

```diff
@@ -66,7 +63,7 @@ const ManchetteWithSpaceTimeWrapper = ({
 };
 
 const meta: Meta<typeof ManchetteWithSpaceTimeWrapper> = {
-  title: 'Manchette with SpaceTimeChart/rendering',
+  title: 'Manchette with SpaceTimeChart/useManchetteWithSpaceTimeChart',
```

Comment:
could probably have a better name. what does this story do ?

Reply:
It shows how to use the `useManchetteWithSpaceTimeChart` hook.